spec/policies/package_policy_spec.rb

```diff
@@ -6,61 +6,46 @@
   let(:resource) do
     double(:resource,
       user: double,
-      resource_type: Faker::Lorem.word,
+      resource_type: 'video',
```

Comment:
The grants & policy are bound to the real configured content types in Package, so we can't just make up stuff here.